### ISSUE TITLE
Change default log level back to Info

### ISSUE THREAD

**Issue by nbolton** [synergy/synergy#4856]
We used notify because INFO log level outputted a lot of info that the user didn't need, so please go back to that log level.

e.g. Not very interesting info...

```
[2015-07-07T16:56:29] INFO: screen "cupertino" updated clipboard 1
[2015-07-07T16:56:29] INFO: switch from "allen" to "cupertino" at 0,551
[2015-07-07T16:56:29] INFO: leaving screen
[2015-07-07T16:56:29] INFO: switch from "cupertino" to "allen" at 1918,661
[2015-07-07T16:56:29] INFO: entering screen
[2015-07-07T16:56:29] INFO: switch from "allen" to "cupertino" at 0,561
[2015-07-07T16:56:29] INFO: leaving screen
[2015-07-07T16:56:29] INFO: switch from "cupertino" to "allen" at 1891,665
[2015-07-07T16:56:29] INFO: entering screen
[2015-07-07T16:56:36] INFO: switch from "allen" to "cupertino" at 0,582
[2015-07-07T16:56:36] INFO: leaving screen
```

**Comment by XinyuHou** [comment]
@nbolton 
The default log level was info before.

**Comment by XinyuHou** [comment]
Resolved by f9b596aa6d653d2a711bd12cc14646d54fb031b0
